02-localproject.Rmd

```diff
@@ -343,21 +345,98 @@ As mentioned earlier, a .gitignore file was created with our R Project. You can
+* <img src="www/images/chapter2_RStudio_gitpaneCommit.png" alt="Git commit icon" height="16"/> Also opens the [RStudio review changes window](#branches-committing-and-pushing);
+* <img src="www/images/chapter2_RStudio_gitpanePull.png" alt="Git pull icon" height="16"/> Perform a pull;
+* <img src="www/images/chapter2_RStudio_gitpanePush.png" alt="Git push icon" height="16"/> Push the current commits;
+* <img src="www/images/chapter2_RStudio_gitpaneHistory.png" alt="Git history icon" height="16"/> Open the [RStudio review changes window](#branches-committing-and-pushing), defaulting to the History panel;
+* <img src="www/images/chapter2_RStudio_gitpaneProjectSettings.png" alt="Project seeings icon" height="16"/> Open the project settings window;
+* <img src="www/images/chapter2_RStudio_gitpaneNewBranch.png" alt="New branch icon" height="16"/> Create a new branch;
+* <img src="www/images/chapter2_RStudio_gitpaneSelectBranch.png" alt="Current branch" height="16"/> Shows the current branch and allows the user to switch branches via the drop down list;
+* <img src="www/images/chapter2_RStudio_gitpaneRefresh.png" alt="Refresh icon" height="16"/> Refresh the git pane preview.
+
+#### Menu bar access
+
+To access the git functionality via the menu bar, navigate to `Tools` > `Version control`. You should then see the options `Commit...`, `Pull Branches`, `Push Branch`, `Hsitory` and `Project Setup...`
+
+![RStudio - Git in the menu bar](www/images/chapter2_RStudio_MenuBarAccess.PNG)
+
+#### Branches, committing and pushing
+
+The bulk of any adding, committing and pushing activities using the RStudio GUI is done using the Review changes window. This can be opened either by selecting `File` > `Commit...` from the RStudio menu bar, or clicking `Diff`, `Commit` or <img src="www/images/chapter2_RStudio_gitpaneHistory.png" alt="Git history icon" height="16"/> in the RStudio git pane.
+
+The Review Changes window consists of 3 panes, the top left shows a preview of all files that have been added, modified or deleted, the top right pane allows you to add a commit message and the last pane shows the git diff result for any file selected in the file preview pane.
+
+![RStudio Review changes window](www/images/chapter2_RStudio_ReviewChanges.PNG)
+
+To make a commit, review the changes in the diff pane by selecting each file in turn and click the Stage button in the Review changes menu bar (either with mulitple/all files selected at once or eac file selected indivudually). Then add your commit message and click the `Commit` button below the Commit message entry box.
+
+Finally, you can just click the `Push` button above the commit message entry box to sync your changes to the remote repository.
+
+You can switch branches by clicking on the current branch name in the Review changes menu bar, i.e. next to the History button.
```

Comment:
Maybe a final paragraph to sum up the advantages/disadvantages of using RStudio GUI. What can't be done via the GUI?

Reply:
Added a short paragraph along these lines. Let me know if you think it needs anything extra.